### PR TITLE
Adding 3 packages mmand_dissutils_geometa

### PR DESCRIPTION
**Pull request by Marie59**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Marie59** [comment]
@conda-forge/help-r, these 3 packages (my first try on conda forge, I hope it is okay) are ready for review !

**Comment by yvanlebras** [comment]
Hmm any helper can give us some info on why there is issues with these recipes on the tests for each recipe? It seems the env don't success to load the R library as here for geometa:
```
2022-05-17T07:30:53.1394763Z > library('geometa')
2022-05-17T07:30:53.1396467Z Error in library("geometa") : there is no package called ‘geometa’
2022-05-17T07:30:53.1397269Z Execution halted
2022-05-17T07:30:53.3980145Z Tests failed for r-geometa-0.6_6-r40hc72bb7e_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
2022-05-17T07:30:53.3981963Z WARNING:conda_build.build:Tests failed for r-geometa-0.6_6-r40hc72bb7e_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
2022-05-17T07:30:53.4380918Z TESTS FAILED: r-geometa-0.6_6-r40hc72bb7e_0.tar.bz2
```

But locally, testing the recipes seem to be ok...

I see this warnings on the tests:
```
2022-05-17T07:30:34.6113422Z source tree in: /home/conda/staged-recipes/build_artifacts/r-geometa_1652772561087/work
2022-05-17T07:30:34.9639630Z INFO:conda_build.build:Packaging r-geometa
2022-05-17T07:30:34.9641514Z Packaging r-geometa
2022-05-17T07:30:34.9915573Z INFO:conda_build.build:Packaging r-geometa-0.6_6-r40hc72bb7e_0
2022-05-17T07:30:34.9917956Z Packaging r-geometa-0.6_6-r40hc72bb7e_0
2022-05-17T07:30:35.0648126Z No files or script found for output r-geometa
2022-05-17T07:30:35.0650204Z WARNING:conda_build.build:No files or script found for output r-geometa
2022-05-17T07:30:35.2875804Z The install/build script(s) for r-geometa deleted the following files (from dependencies) from the prefix:
2022-05-17T07:30:35.2878121Z ['x86_64-conda-linux-gnu/lib/libstdc++.so.6.0.28']
2022-05-17T07:30:35.2879514Z This will cause the post-link checks to mis-report. Please try not to delete and files (DSOs in particular) from the prefix
2022-05-17T07:30:35.2880988Z WARNING:conda_build.build:The install/build script(s) for r-geometa deleted the following files (from dependencies) from the prefix:
2022-05-17T07:30:35.2882228Z ['x86_64-conda-linux-gnu/lib/libstdc++.so.6.0.28']
2022-05-17T07:30:35.2883466Z This will cause the post-link checks to mis-report. Please try not to delete and files (DSOs in particular) from the prefix
2022-05-17T07:30:35.5020506Z number of files: 0
2022-05-17T07:30:35.7058737Z Fixing permissions
2022-05-17T07:30:35.8745645Z Packaged license file/s.
2022-05-17T07:30:35.8748056Z INFO :: Time taken to mark (prefix)
2022-05-17T07:30:35.8749452Z         0 replacements in 0 files was 0.00 seconds
2022-05-17T07:30:36.0034777Z Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
2022-05-17T07:30:36.0039936Z WARNING:conda_build.build:Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
```

Is there any issue on the test env or in our recipes ? Don't hesitate to comment ;) !

**Comment by dbast** [comment]
This is missing the required build.sh/bld.bat files also created by the skeleton and to be added to the PR... also the R recipe folder names should be prefixed with "r-" same as the package names.

**Comment by yvanlebras** [comment]
Thank you so much Daniel for this fast and detailled reply! Marie will go on it!

File: recipes/r-mmand/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-rcpp
```
Comment (dbast) [inline recipes/r-mmand/meta.yaml:41]:
```suggestion
    - r-rcpp
    - llvm-openmp  # [osx]
    - libgomp      # [linux]
```

**Comment by dbast** [comment]
win_64 builds fail for r-geometa, which could be ignored as the recipe is noarch. But the win build stops due to that and does not even do a build attempt for r-mmand. Please either but r-geometa into a separate PR or event split the PR into one PR for each recipe.... Each recipe so far looks pretty good!

**Comment by Marie59** [comment]
@dbast I will do the 3 PR ! Thanks a lot for your help !

**Comment by Marie59** [comment]
@dbast yes totally we can close it !

**Pull request closed without merging**